packages/shared/lib/contexts/onboarding/actions/index.ts

```diff
@@ -14,6 +14,7 @@ export * from './syncShimmerClaimingAccount'
 export * from './generateMnemonicForOnboardingProfile'
 export * from './handleTransactionInclusionEventForShimmerClaiming'
 export * from './initialiseFirstShimmerClaimingAccount'
+export * from './initialiseOnboarding'
```

Comment:
Perhaps this file should be initialiseOnboardingFlow as it is going to navigate the user to the start of the onboarding flow.